topics/ui/controls/tabs.md

```diff
@@ -24,49 +25,71 @@ Follow the rules in [Groups of controls](groups_of_controls.md).
 
 ## How to use
 
-Use title capitalization for tab labels.
+### Label
 
-Make the label short, preferably no more than 3 words.
+Use title capitalization for tab labels. Make the label short, preferably no more than 3 words.
 Avoid generic words, such as "General" or "Advanced".
 See [Writing short and clear](writing_short.md).
 
-![](02_naming.png){width=284}
+<table style="none" border="false">
+  <tr>
+<td width="50%">
+      <format color="Green" style="bold">Correct</format><img src="02_How_to_use_correct.png" alt="A correct example of a tab label"/>
+    </td>
+    <td width="50%">
+      <format color="Red" style="bold">Incorrect</format><img src="02_How_to_use_incorrect.png" alt="An incorrect example of a tab label"/>
+    </td>
+  </tr>
+</table>
+
```

Comment:
Possibly add more mistakes to the Incorrect version? It appears under a paragraph with several rules but illustrates only one about Advanced.